p2p/src/delegates.rs

```diff
@@ -122,19 +135,295 @@ impl ProposalVerifier for DummyDelegate {
 impl GreaseInitializer for DummyDelegate {
     async fn generate_initial_proofs(
         &self,
-        _in: Comm0PrivateInputs,
+        input_public: Comm0PublicInputs,
+        input_private: Comm0PrivateInputs,
         metadata: &ChannelMetadata,
     ) -> Result<Proofs0, DelegateError> {
         info!("DummyDelegate: Generating initial proofs for {}", metadata.channel_id().name());
-        Ok(Proofs0::default())
+
```

Comment:
Please move all this into a separate struct in another module (e.g. `mod bjj_delegate`) that implements the necessary traits.

`DummyDelegate` is meant for testing and so must be super fast, and return immediately.